docs/application-connector/docs/assets/connectorapi.yaml

```diff
@@ -97,6 +97,12 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/csrResponse'
+        '400':
+          description: 'Invalid CSR.'
```

Comment:
The CSR may be actually valid, it just does not meet our subject fields requirements, I would suggest doing something like 'Improper subject values in CSR', you may want to double-check it with @lszymik